bpf/lib/common.h

```diff
@@ -336,25 +336,29 @@ struct proxy4_tbl_key {
 	__u16 dport; /* dport must be in front of sport, loaded with 4 bytes read */
 	__u16 sport;
 	__u8 nexthdr;
+	__u8 pad;
```

Comment:
The proxy key is essentially a full 5-tuple, since destination address is left out only because it's always the same? If so, it would be cleaner if the key is the full 5-tuple, if we'll be doing lookups in these maps from external processes in the future.